### PR TITLE
MAPI-1330 - enabled Dynamo in topic registrations stats

### PR DESCRIPTION
**Pull request by agulowaty**

### CODE REVIEW

**Comment by agulowaty** [comment]
@davidfurey @alexduf @DiegoVazquezNanini

File: common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala [hunk 1]
```diff
@@ -13,14 +14,14 @@ import scala.collection.JavaConversions._
 
 class DynamoTopicSubscriptionsRepository(client: AsyncDynamo, tableName: String)
   (implicit ec: ExecutionContext) extends TopicSubscriptionsRepository {
-
+  val logger = Logger(classOf[DynamoTopicSubscriptionsRepository])
   val TopicIdField = "topicId"
   val TopicSubscriberCountField = "topicSubscriberCount"
 
-  override def subscribe(topic: Topic): Future[RepositoryResult[Unit]] =
+  override def deviceSubscribed(topic: Topic): Future[RepositoryResult[Unit]] =
```
Comment (DiegoVazquezNanini) [inline common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala]: Is it necessary to rename these as part of this pull request, or would it be better to make a new PR instead?

Comment (agulowaty) [inline common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala]: should I create separate PR just for renaming ?

Comment (DiegoVazquezNanini) [inline common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala]: Yes, please. 
Can you please explain why you want to do this renaming?

Comment (agulowaty) [inline common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala]: Naming was wrong in that sense `TopicSubscriptionsRepository` is not responsible for subscribing to topics (as methods suggested). I wanted to emphasise fact it is `notified` on topic subscription instead.

Comment (DiegoVazquezNanini) [inline common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala]: Ok, thanks for the clarification.

**Comment by agulowaty** [comment]
any comments ?

**Comment by DiegoVazquezNanini** [comment]
:+1: LGTM

**Pull request merged**